### DISCUSSION TITLE
Issue with Full Screen in Zoom Call

### DISCUSSION THREAD

**Discussion by sibuluq** [elementary/triage#126] [Q&A] [answered]
I am not sure if this happens with other applications, but I have seen it happen in Zoom. Whenever I am in a call and go into a full screen view, the application goes into full screen mode, but only takes up like 1/4 of the top left of the screen and becomes unresponsive until I move it to a different workspace, where it is no longer in full screen mode.

Originally posted at [StackExchange](https://elementaryos.stackexchange.com/questions/26032/issue-with-full-screen-in-zoom-call).

**Comment by sibuluq** [comment]
> I believe it is a zoom bug not elementary issue, it happens to all my devices using linux in general and teams or skype does not make the same issue, just enter full screen from elementary itself not from zoom (search shortcuts for full screen).
> 
> Answer by [Omar Shawky](https://elementaryos.stackexchange.com/users/22395/omar-shawky) at [StackExchange](https://elementaryos.stackexchange.com/a/26039).

**Comment by hanaral** [comment, marked as answer]
I'm afraid as you said yourself, this is a Zoom issue and isn't under control of elementary. If you have any specific glitches to do with the compositor, open them under https://github.com/elementary/gala/issues/

**Reply by sibuluq** [reply]
I think there is a misunderstanding here.

I'm only trying to participate in migrating questions from StackExchange to here because of the tweet https://twitter.com/DanielFore/status/1341114091876728832. So this is neither my question nor my answer.

Sorry for causing a misunderstanding.

**Reply by hanaral** [reply]
Apologies, I should've realised you were trying to help.
I think the issue is that a lot of these aren't answerable let alone actionable to begin with :(
